packages/OmegaPrint-Core.package/OPPrinter.class/instance/value..st

```diff
@@ -0,0 +1,25 @@
+value
+value: inputNode
+
+	| selectors ruleName ruleMessageName aNode result |
+	inputNode isOhmMatchResult 
+		ifTrue: [aNode := inputNode cst]
+		ifFalse: [aNode := inputNode ].
+	
+	ruleName := aNode ruleName asSymbol.
+	ruleMessageName := self class ruleNameToMessageName: ruleName.
+	
+	selectors := self selectors select: [:msg | msg beginsWith: ruleMessageName].
+	self assert: selectors size <= 1.
+
+	[ result := self value: aNode skippedSpacesNodes ]
```

Comment:
da wir in der Variable tatsaechlich nur kommentare und spaces haben wuerde ich sie glaube ich nicht `result` nennen, vielleicht eher sowas wie `formattedWhitespace`, bin da gerne offen fuer Vorschlaege

Reply:
passt dir auch einfach nur `spaces`?

Reply:
passt mir :+1: